src/auth/SignUp.jsx

```diff
@@ -59,22 +60,24 @@ export const SignUp = () => {
     return undefined;
   };
 
-  const handleInputs = (e) =>
+  const handleInputs = (e) => {
     setNewUser({ ...newUser, [e.target.name]: e.target.value });
+    setErrorClient(null);
+    dispatchUser({ type: ERROR, payload: null });
+  };
 
   return (
     <SignUpStyles>
       <GridAuth height='800px'>
         <FormStyles>
-          <SignUpGreetingStyles>
-            <Title>
-              Hurrraaaaay! <br />
-              Lets us know who you are!
-            </Title>
-            <PrimaryText>
-              We won&apos;t share your info with anybody. I promise
-            </PrimaryText>
-          </SignUpGreetingStyles>
+          <Title>
```

Comment:
Now the headline is sitting on the top again.

Reply:
Weird, This is how it looks in my monitor:
![image](https://user-images.githubusercontent.com/58083933/111911653-bc934f00-8a66-11eb-8ef1-7cb55a940ca9.png)

That can be handle if want to make it responsive then.

Reply:
when I am in branch error-handler and start it it sits on the top again. that is weird then,

Reply:
You also have a big monitor, right?

Reply:
the only monitor size where it has a little distant to the top is on the notebook maybe 20px....

Reply:
yes my i have a big monitor